### PR TITLE
Add spec for get measures

### PR DESCRIPTION
**Pull request by luizbranco**
@tim-speed can you review the descriptions too?

### CODE REVIEW

File: specs/provider.yaml [hunk 1]
```diff
@@ -913,6 +913,43 @@ paths:
...
+
+  /resources/{id}/measures:
+    get:
+      summary: Get how much a resource has used its features
+      description: |
+        Manifold will call this endpoint daily to get usage information about a
+        resource and its features.
+
+        The Provider should only need to hold information about the time left
+        in the current month and the previous month.
```
Comment (tim-speed) [inline specs/provider.yaml]: Add something similar to this which defines how the billing months are defined:
`Months are defined by the Manifold billing period which starts at the first of each month in UTC-0 and ends at the start of the next.`

File: specs/provider.yaml [hunk 2]
```diff
@@ -913,6 +913,43 @@ paths:
...
+          required: true
+          type: string
+          format: datetime
+      tags:
+        - Resource
+      responses:
+        200:
+          description: |
+            A response should contain a measures map in which the key represents a
+            resource feature and the value is the most updated usage value of the
```
Comment (tim-speed) [inline specs/provider.yaml]: Add something like:
`This usage value is assumed to be the sum total of usage for that feature for the billing period`

File: specs/provider.yaml [hunk 3]
```diff
@@ -1043,3 +1080,33 @@ definitions:
     additionalProperties: false
     required:
     - message
+
+  ResourceMeasures:
+    type: object
+    description: |
+      Wrapper object for resource measures body.
```
Comment (tim-speed) [inline specs/provider.yaml]: We don't need to worry about wrapper objects for the provider API, it's not a standard we have for them, only one we have internally:
https://docs.manifold.co/providers#tag/Credential
No need to get them to do the extra work.

File: specs/provider.yaml [hunk 4]
```diff
@@ -1043,3 +1080,33 @@ definitions:
...
+      body:
+        $ref: '#/definitions/ResourceMeasuresBody'
+    additionalProperties: false
+    required:
+      - body
+
+  ResourceMeasuresBody:
+    type: object
+    description: |
+      Object used to hold information about a resource and its features usage.
```
Comment (tim-speed) [inline specs/provider.yaml]: If you wanted to reword:
"Object describing a resource's feature usage for a billing period."

File: specs/provider.yaml (hunk 3 above)
Comment (tim-speed) [reply specs/provider.yaml]: I think it's important to remove this, but let me know if you think otherwise.

Comment (luizbranco) [reply specs/provider.yaml]: Makes sense.

**Comment by tim-speed** [comment]
:sheep: it

**Pull request merged**